devpack/src/main/java/io/neow3j/devpack/compiler/NeoVariable.java

```diff
@@ -0,0 +1,16 @@
+package io.neow3j.devpack.compiler;
+
+import org.objectweb.asm.tree.LocalVariableNode;
+
+public class NeoVariable {
```

Comment:
No other method is required?

Reply:
ah, just accessible by the package and stuff. 😄

Reply:
I did not add any getters and access the fields directly for convenience. These Neo<something> classes will probably only be used internally by the devpack.